### PR TITLE
[Docs] Add clarifying note on PermutationMNK 

### PR DESCRIPTION
**Pull request by imaginary-person**
Added a clarifying note that for PermutationMNK, each mode's shape & strides are listed in the order of `the outermost dim to the innermost dimension`.

`(4, 4, 2):(1, 8, 4)` can perhaps be more easily understood by folks familiar with numpy semantics if they were aware of the order of shape & strides (since it's opposite to the numpy convention).

To elaborate, the visual example in the doc on scatter permutation on M-mode can also be visualized as:

```python
import torch
before = torch.arange(32).view(2, 4, 4).contiguous()
after = a.as_strided(size=(2, 4, 4), stride=(4, 8, 1))
# view indices before permutation 
print(before.flatten())
# view indices after permutation 
# If the value at ith index is n, that means after[i] = before[n]
print(after.flatten())
```

### CODE REVIEW

**Comment by imaginary-person** [comment]
Hi @thakkarV, can you please help review this change, as per your convenience?
 It'd help me confirm if I understand PermutationMNK correctly. 
Thanks in advance!

**Comment by ccecka** [comment]
I don't see this adding value to the current explanation with the pre- and post- permutation coordinates listed explicitly. Does "innermost" mean fastest changing and on-the-left? That doesn't appear to be your meaning. Point is, you would also need to define these terms.

The description of `Layout`, the colexicographical ordering of coordinates, and the intuition of reading `Layout`s from left to right is given in other sections.

**Comment by imaginary-person** [comment]
Hi @ccecka, thanks a lot for the info!

By innermost dimension, I meant the fastest changing dimension, which is the leftmost in case of cute, but the rightmost in numpy semantics, which may cause some confusion to folks new to cutlass internals.

> The description of Layout, the colexicographical ordering of coordinates, and the intuition of reading Layouts from left to right is given in other sections.

 Thanks, I'll go through the rest of the documentation.
It looks like this change would be redundant.
